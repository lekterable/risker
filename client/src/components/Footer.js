import React from 'react'
import './Footer.css'

export default props => {
	return (
		<footer className="footer text-center">
			<div className="container">
				<span className="text-muted">{props.content}</span>
			</div>
		</footer>
	)
}
