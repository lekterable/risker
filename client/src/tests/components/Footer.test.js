import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../../components/Footer'

describe('Footer', () => {
	it('should match snapshot', () => {
		const wrapper = shallow(<Footer />)

		expect(wrapper).toMatchSnapshot()
	})
})
