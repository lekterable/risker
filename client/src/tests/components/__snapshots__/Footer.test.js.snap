// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer
  className="footer text-center"
>
  <div
    className="container"
  >
    <span
      className="text-muted"
    />
  </div>
</footer>
`;
